FAST-LIO/src/IMU_Processing.hpp

```diff
@@ -194,10 +194,12 @@ void ImuProcess::IMU_init(const MeasureGroup &meas, esekfom::esekf<state_ikfom,
   init_state.grav = S2(- mean_acc / mean_acc.norm() * G_m_s2);
   const auto& orientation = meas.imu.front()->orientation;
   // 如果orientation消息可用的话,以IMU的orientaion作为初始姿态
-  if( orientation.x != 0 || orientation.y != 0 || orientation.z != 0 || orientation.w != 0 ){
-    Eigen::Quaterniond qua(orientation.w, orientation.x, orientation.y, orientation.z);
-    init_state.rot = qua;
-  }
+  // if( orientation.x != 0 || orientation.y != 0 || orientation.z != 0 || orientation.w != 0 ){
```

Comment:
Let's get in the practice of not leaving behind commented out code. Either delete it or add a TODO if we might want to bring it back or dive deeper later!

Reply:
Makes sense, added a TODO